.buildkite/integration.pipeline.yml

```diff
@@ -201,3 +201,8 @@ steps:
     depends_on:
       - int-packaging
     command: "buildkite-agent pipeline upload .buildkite/bk.integration.pipeline.yml"
+
+  - label: "Triggering custom ECH integration tests"
```

Comment:
Instead of ECH let's use FIPS or FRH to be consistent?  Maybe we should rename the pipeline file to be consistent as well?